test/integration/acceptance/queries_acceptance_test.cpp

```diff
@@ -0,0 +1,466 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <gtest/gtest.h>
+
+#include "backend/protobuf/transaction.hpp"
+#include "framework/integration_framework/integration_test_framework.hpp"
+#include "framework/specified_visitor.hpp"
+#include "integration/acceptance/acceptance_fixture.hpp"
+#include "interfaces/permissions.hpp"
+#include "utils/query_error_response_visitor.hpp"
+
+using namespace integration_framework;
+using namespace shared_model;
+
+class QueriesAcceptanceTest : public AcceptanceFixture {
+ public:
+  QueriesAcceptanceTest()
+      : itf(1),
+        invalidPrivateKey(kUserKeypair.privateKey().hex()),
+        invalidPublicKey(kUserKeypair.publicKey().hex()) {
+    /*
+     * It's deliberately get broken the public key and privite key to simulate a
```

Comment:
More correct sentence: "deliberately break the public and private keys to simulate..."